docs/index.md

```diff
@@ -12,6 +12,37 @@ Use the navigation to the left to read about the available resources.
 
 ## Example Usage
 
+Terraform 0.13+ uses the Terraform Registry:
+```hcl
+terraform {
+  required_providers {
+    datadog = {
+      source  = "DataDog/datadog"
+      # version = "~> 1.0"
+    }
+  }
+}
+
+# Configure the Datadog provider
+provider "datadog" {
+  api_key = "${var.datadog_api_key}"
+  app_key = "${var.datadog_app_key}"
```

Comment:
Can you take the chance to use the new syntax without `"${}"` ?

Reply:
I tried with terraform 0.14, the variables themselves are defined the same so I didn't include it here, but it works to reference them without all the `${}`